namespace-provisioner/customize-installation.md

```diff
@@ -352,11 +351,11 @@ Options if using GitOps
 
```

Comment:
line 350, the link for private git repo was already referenced in line 321, should it go twice?

Reply:
We have this mentioned at two places in the referencing document as well